tests/resources/rules/rules.json

```diff
@@ -105018,13 +105044,17 @@
                             {
                                 "dataset": "lb.xpt",
                                 "domain": "LB",
-                                "execution_status": "execution_error",
-                                "execution_message": "SQL operator execution error",
+                                "execution_status": "success",
```

Comment:
Looks like this case might be wrong, because the old engine doesn't have any errors